api/src/db/migrations/v149_add_environment_data.ts

```diff
@@ -102,7 +102,7 @@ export interface v149ProfileData extends v149IndustrySpecificData {
   naicsCode: string;
   foreignBusinessTypeIds: v149ForeignBusinessTypeId[];
   nexusDbaName: string;
-  needsNexusDbaName: boolean;
+  needsNexusDbaName: boolean | undefined;
```

Comment:
Added due to typescript error in v150. unable to use delete keyword since needsNexusDbaName expected to be a boolean/exist

Reply:
TS2790: The operand of a delete operator must be optional.